### ISSUE TITLE
Not forward-compatible API data geometries

### ISSUE THREAD

**Issue by chahank** [CLIMADA-project/climada_python#764]
When upgrading to shapely 2.1, which is highly recommended for computational efficiency, all data on the API that was created with a version shapely < 2.0 will not be compatible with the unsequa module due to pickling changes.

UserWarning: Unpickling a shapely <2.0 geometry object. Please save the pickle again; shapely 2.1 will not have this compatibility.

https://github.com/shapely/shapely/issues/1613

Simple solution: open and save all API objects with shapely content (all geodataframes with geometries) using shapely 2.0 or higher.

**Comment by peanutfun** [comment]
@chahank @emanuel-schmid Was this resolved for the data on the API?

**Comment by chahank** [comment]
I do not know. @emanuel-schmid ?

**Comment by emanuel-schmid** [comment]
> Was this resolved for the data on the API?

No, not yet. Perhaps they just got more tolerant towards older versions?
